x-pack/platform/plugins/shared/fleet/server/types/rest_spec/agent.ts

```diff
@@ -39,12 +39,49 @@ export const GetAgentsRequestSchema = {
       getStatusSummary: schema.boolean({ defaultValue: false }),
       sortField: schema.maybe(schema.string()),
       sortOrder: schema.maybe(schema.oneOf([schema.literal('asc'), schema.literal('desc')])),
+      searchAfter: schema.maybe(schema.string()),
```

Comment:
reposting from PR description:

@elastic/fleet I couldn't get the Kibana request schema to accept a multi-part query param and convert it to an array... I think this would be better, please let me know if you know how to get that to work 🙏

changing this to `schema.maybe(schema.arrayOf(schema.any()))` and passing params like `&searchAfter=123&searchAfter=test` or even `&searchAfter[]=123&searchAfter[]=test` didn't work. it only returned error about `searchAfter` not matching expected schema

Reply:
it seems when using an array of any it do not parse correctly number or string, (it seems expected as there not really way to get that), so I guess it's okay to use a string here I am wondering if it will make the API easier to use if we add a meta to agent list response something like `nextSearchAfter: "[stringified...]"`  so user will not have to know about getting the last sort field of the agents, WDYT?

Reply:
I think this is a good idea, I will take a look

Reply:
@nchaulet added in 0783feb and updated PR description. it helped writing the API tests a little easier too :)